example.py

```diff
@@ -0,0 +1,81 @@
+# RVR drive example 2022-02-17
+
+import board
+import busio
+import time
+import math
+
+import adafruit_hcsr04
+from sphero_rvr import RVRDrive
+
+#sonar = adafruit_hcsr04.HCSR04(trigger_pin=board.IO5, echo_pin=board.IO4)   # lilygo ESP32-S2
+#sonar = adafruit_hcsr04.HCSR04(trigger_pin=board.B1, echo_pin=board.B0)     # blackpill
+#sonar = adafruit_hcsr04.HCSR04(trigger_pin=board.GP10, echo_pin=board.GP11) # RP Pico
+sonar = adafruit_hcsr04.HCSR04(trigger_pin=board.D11, echo_pin=board.D10)    # M4 Metro express
+
+#rvr = RVRDrive(uart = busio.UART(board.IO1, board.IO2, baudrate=115200))    # lilygo ESP32-S2
+#rvr = RVRDrive(uart = busio.UART(board.A2, board.A3, baudrate=115200))      # blackpill
+#rvr = RVRDrive(uart = busio.UART(board.GP4, board.GP5, baudrate=115200))    # RP Pico
+rvr = RVRDrive(uart = busio.UART(board.D1, board.D0, baudrate=115200))       # M4 Metro express

```

Comment:
This can be copied and pasted into the head of other projects - and just uncommented for the respective board. It works equally on these 4 different boards with 4 different CPUs (samd51, stm32f411, rp2040 and esp32-s2)